etc/kayobe/inventory/group_vars/cis-hardening/cis

```diff
@@ -31,6 +31,9 @@ rhel9cis_rule_1_9: false
 # Disable requirement for password when using sudo
 rhel9cis_rule_5_3_4: false
 
+# Disable shell inactivity timeout
+rhel9cis_rule_5_6_3: false
```

Comment:
I guess the issue here is we slowly reduce our CIS score over time by disabling these things. Anyone else have thoughts?